engine/src/main/java/org/hawkular/alerts/engine/impl/ispn/IspnAlertsServiceImpl.java

```diff
@@ -80,8 +80,8 @@
     boolean saveThinAlerts = false;
 
     public void init() {
-        alertsLifespanInHours = ConfigProvider.getConfig().getValue("engine.backend.ispn.alerts-lifespan", Long.class);
         eventLifespanInHours = ConfigProvider.getConfig().getValue("engine.backend.ispn.events-lifespan", Long.class);
+        alertsLifespanInHours = ConfigProvider.getConfig().getValue("engine.backend.ispn.alerts-lifespan", Long.class);
```

Comment:
I assume we want to check if `engine.backend.ispn.events-lifespan` is the next fail?

Reply:
Yes, we want to check if the issue affects all configuration values or only `engine.backend.ispn.alerts-lifespan`.